src/wormhole_mailbox_server/server_tap.py

```diff
@@ -29,16 +29,22 @@ class Options(usage.Options):
         ("motd", None, None, "Send a Message of the Day in the welcome"),
         ]
     optFlags = [
-        ("disallow-list", None, "refuse to send list of allocated nameplates"),
+        ("disallow-list", None, "default. refuse to send list of allocated nameplates"),
+        ("allow-list", None, "allow to send list of allocated nameplates"),
         ]
 
     def __init__(self):
         super(Options, self).__init__()
+        # Default values
         self["websocket-protocol-options"] = []
-        self["allow-list"] = True
+        self["allow-list"] = False
```

Comment:
I don't really want to flip around the CLI args like this -- it breaks everyone's command-lines with very little gain.

Instead we could suggest to operators that they should prefer to use `--disallow-list` in the immediate term, and that clients should not build in functionality that depends on `list` working all the time.

Reply:
Um, not sure how this may break command lines? This was not my intention

Reply:
This flips things around to turn `list` off by default. So if operators already wanted `list` to work, they'll have to change their command-lines (since the default changes).

Reply:
Well yes, that's how "enabling/disabling a feature by default" works? I wouldn't consider that breaking, but I can change it back if you don't like it

Reply:
How about this: I keep the current default value, but also add the `allow-list` flag and print a warning that one should use it to keep that behavior in the future.

Reply:
Yeah, lets leave the flipping around of CLI flags alone in this PR. There's already a way to run with `list` on, and a way to run with `list` off.

(The CLI is the only public interface here, so I don't see a lot of gain in changing it merely to communicate "you should probably turn off `list` by default").